app/views/sidebar/_question.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <%= render :partial => "like/like", :locals => {:node => @node, :tagnames => @tagnames } %>
 
-  <a style="margin-top:-16px;margin-bottom:0;" class="btn btn-sm btn-block btn-link d-lg-none" href="javascript:void()" onClick="toggle_sidebar()"><i class="fa fa-chevron-down"></i></a>
+  <a style="margin-top:-16px;margin-bottom:0;" class="btn btn-sm btn-block btn-link d-lg-none" href="javascript:void()" onClick="toggle_sidebar()"><i class="fa fa-chevron-down" style="font-size:26px; padding-top:15px; padding-bottom:15px;"></i></a>
```

Comment:
Here's how it looks:

![code-review-changes-1](https://user-images.githubusercontent.com/70844041/158197922-0845dc2d-c995-49da-be97-526cac089ac3.PNG)

Reply:
This looks perfect @AbihaFatima. Thank you!

Reply:
Also instead of using styles, why not use bootstrap classes? It's better to maintain consistency.

Reply:
Sure!